sys/kern/build.rs

```diff
@@ -80,14 +401,14 @@ fn generate_statics() -> Result<()> {
     /////////////////////////////////////////////////////////
     // Basic constants and empty space
 
-    let task_count = kconfig.tasks.len();
+    let task_count = gen.tasks.len();
```

Comment:
This is the length of the `TokenStream`, which seems... odd.

Reply:
oh wait, never mind

Reply:
It is the length of the `Vec<TokenStream>` where we generate one stanza per task.